website/docs/r/storage_data_lake_gen2_filesystem.html.markdown

```diff
@@ -26,6 +26,8 @@ resource "azurerm_storage_account" "test" {
   location                 = azurerm_resource_group.test.location
   account_tier             = "Standard"
   account_replication_type = "LRS"
+  account_kind             = "StorageV2"
+  is_hns_enabled           = "true"
```

Comment:
Would it also make sense to add a note to the `storage_account_id` id property detailing this?

Reply:
Good idea. I updated the PR to reflect this